plugins/time-saver-backend/src/database/TimeSaverDatabase.ts

```diff
@@ -605,7 +614,7 @@ export class TimeSaverDatabase implements TimeSaverStore {
           this.db.raw(`DATE(created_at) as date`),
           this.db.raw('SUM(time_saved) as total_time_saved'),
         )
-        .groupBy('template_name', 'date');
+        .groupBy('team', 'date');
```

Comment:
here is the change with a logic, should by grouped by template_name.

Reply:
Reverted